cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -5,7 +5,7 @@ namespace: default
 labels:
   k8s-app: skydns
 desiredState:
-  replicas: {DNS_REPLICAS}
+  replicas: {{ pillar['dns_replicas'] }}
```

Comment:
Those assume that the salt stuff is done - it's more hand-holding for
launching the skydns pods.  I'd be happy ot see that bit removed, with a
reference to "and then we salt it and the pod gets scheduled"

On Tue, Jan 20, 2015 at 9:45 AM, Zach Loafman <notifications@github.com>
wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601#discussion-diff-23239852>
> :
>
> > @@ -5,7 +5,7 @@ namespace: default
> >  labels:
> >    k8s-app: skydns
> >  desiredState:
> > -  replicas: {DNS_REPLICAS}
> > +  replicas: {{ pillar['dns_replicas'] }}
>
> Hm. Are you talking about `cluster/addons/dns/README.md`? Those
> instructions don't look right ... they don't include any instructions on
> how you'd bonk salt to update the minions for the `DNS_SERVER_IP`. They
> look pre-ENABLE_CLUSTER_DNS, pre-salt config, at the very least. At the
> moment, the configuration seems pretty static once you create the cluster,
> unless you know the right incantations. Given that, would you mind a
> separate pull for cleanup on the docs?
> ... <#14b0876ce70e1c20_>
> On Tue, Jan 20, 2015 at 9:26 AM, Tim Hockin <notifications@github.com>
> wrote: In cluster/addons/dns/skydns-rc.yaml.in <
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3601#discussion-diff-23238511>
> : > @@ -5,7 +5,7 @@ namespace: default > labels: > k8s-app: skydns >
> desiredState: > - replicas: {DNS_REPLICAS} > + replicas: {{
> pillar['dns_replicas'] }} There is a doc that explain to mortal users how
> DNS works. Needs updates? -- Reply to this email directly or view it on
> GitHub <
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3601/files#r23238511>
> .
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601/files#r23239852>
> .
>